omod/src/main/resources/config.xml

```diff
@@ -233,5 +233,29 @@
         <description>Able to access the find patient app</description>
     </privilege>
```

Comment:
@slubwama Why are coreapps privileges included in this config.xml? They are already added as part of coreapps